eu/ja-jp/content/topics/end-user/device-access/macos-mfa-end-user.htm

```diff
@@ -294,34 +294,34 @@ <h2><span class="mc-variable okta-variables.ProductName variable">Okta</span>に
                                                             <p>ユーザー名とパスワードを入力します。</p>
                                                         </li>
                                                         <li value="3">
-                                                            <p>デスクトップにアクセスできるようになる前に、<span class="mc-variable okta-feature-names.Okta_Device_Access variable">［Okta Device Access（Oktaデバイスアクセス）］</span>セットアップ画面が表示されます。<span class="uicontrol">［Continue（続行）］</span>をクリックします。</p>
```

Comment:
@pengyao-okta We have another regression, these UI controls should not have been revised (here and elsewhere)